cdb2jdbc/src/main/java/com/bloomberg/comdb2/jdbc/Comdb2Handle.java

```diff
@@ -13,20 +13,29 @@
    limitations under the License. */
 package com.bloomberg.comdb2.jdbc;
 
-import java.nio.ByteBuffer;
-import java.io.*;
-import java.util.*;
-import java.util.logging.*;
-import javax.net.ssl.*;
+import com.bloomberg.comdb2.jdbc.Cdb2Query.Cdb2BindValue;
```

Comment:
I would very much prefer to import a whole package instead of individual classes in the package. It's cleaner to read and much easier for non-IDE users to modify the code.